src/components/pageElements/CornerSubmission/SubmissionReview.jsx

```diff
@@ -302,14 +309,15 @@ Image.propTypes = {
 };
 
 const AttachmentReview = ({ path }) => {
-  const storage = useStorage();
+  const { storage } = useFirebaseStorage();
   const [data, setData] = useState();
+
   getDownloadURL(ref(storage, path)).then(setData);
 
   return (
     <Card>
       <h4 className="-mt-2 text-lg font-bold">Existing Monument Record Sheet</h4>
-      <div className="h-[400px] max-w-[300px] justify-self-center">
+      <div className="contents h-[400px] max-w-[300px] justify-self-center">
```

Comment:
What does `contents` do? I couldn't find it in the TW docs.

Reply:
I was thinking it was https://developer.mozilla.org/en-US/docs/Web/CSS/contain#content